pandas/core/indexes/category.py

```diff
@@ -655,3 +655,24 @@ def _delegate_method(self, name: str, *args, **kwargs):
         if is_scalar(res):
             return res
         return CategoricalIndex(res, name=self.name)
+
+    def replace(
```

Comment:
Not adding @doc here since it will require a lot of conditionals due to regex being disabled. We can add it after `replace()` is sorted out.